source/transfer_list.rst

```diff
@@ -343,6 +343,23 @@ Inputs:
 
    #. Add `align8(te.hdr_size + te.data_size)` to `te_base_addr`.
 
+
+Adding a void TE
+^^^^^^^^^^^^^^^^
+
+Inputs:
+
+- `te_base_addr`: Base address where void TE to be added
```

Comment:
It might be useful to put the alignment part in here so that it doesn't need to be mentioned in the two calling operations (e.g. call this `start_address`, and then start with calculating `te_base_addr` as `align8(start_address)`)?